<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>681</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_file">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Record File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_newFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Recored File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_leftShank">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>450</y>
      <width>202</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left Shank</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rightShank">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>450</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right Shank</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="trunk" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_trunk">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trunk</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>411</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCustomPlot" name="leftThigh" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="rightThigh" native="true">
       <zorder>leftThigh</zorder>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>480</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCustomPlot" name="leftShank" native="true">
       <zorder>leftAnkle</zorder>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="rightShank" native="true">
       <zorder></zorder>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>411</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_leftThigh">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left Thigh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_rightThigh">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right Thigh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuCalls"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
